### ISSUE TITLE
libgmp on ARM fails at runtime

### ISSUE THREAD

**Issue by skalk** [genodelabs/genode#3068]
When using libgmp (used by g++) while running the `noux_tool_chain_auto.run` script on any ARM platform, it fails at runtime due to a missing symbol:
```
[init -> noux -> /libexec/gcc/arm-none-eabi/6.3.0/cc1plus] Error: LD: jump slot relocation failed for symbol: '__gmpn_invert_limb'
[init -> noux -> /libexec/gcc/arm-none-eabi/6.3.0/cc1plus] Error: Uncaught exception of type 'Linker::Not_found'
[init -> noux -> /libexec/gcc/arm-none-eabi/6.3.0/cc1plus] Warning: abort called - thread: ep
[init -> noux] child /libexec/gcc/arm-none-eabi/6.3.0/cc1plus exited with exit value 1
```

**Comment by cproc** [comment]
Fixed in 99fe5f3.

**Comment by nfeske** [comment]
Thanks. I merged your fix to staging.
